spec/shopify_transporter/exporters/magento/product_exporter_spec.rb

```diff
@@ -345,22 +349,22 @@ module Magento
 
               expected_result = [
                 {
```

Comment:
are all of these changes a result of the change to how we log to stdout?

Reply:
Yes. Everything needs to be stringified all the way down because of the fact that we now print to stdout and then use `JSON.parse` on the data that we grab from stdout. Unfortunately, `deep_stringify_keys` only works on `Hash`. The alternative is to write our own `stringify_data` or something that works on both `Hash` and `Array` even when they're nested in each other. I don't mind doing that if you prefer it.

Reply:
No no this is fine. Just a question. I wouldn't want you to go embark on that just to keep this from changing. This still does its job.